views/js/pciCreator/dev/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -262,29 +262,55 @@ define([
 
             // cache controls states
             _.forEach(this.controls, function(ctr, id) {
-                ctrCache[id] = ctr.getState();
+                self.ctrCache[id] = ctr.getState();
                 ctr.disable();
             });
 
+            if (delayInSeconds > 0) {
+                // init countdown
+                this.initCountdown();
+            }
+
+            if(this.hasMediaStimulus()) {
+                return;
+            }
+
+            this.initDelay();
+        },
+
+        initDelay: function initDelay() {
+            var delayInSeconds = this.config.delayMinutes * 60 + this.config.delaySeconds;
```

Comment:
We don't need this variable because we only use it once.
Or, we can change the scope by putting inside `this.askPermissionAccessMic`